src/lando/utils/management/commands/create_environment_repos.py

```diff
@@ -153,23 +151,8 @@
             "required_permission": SCM_CONDUIT,
         },
     ],
-    Environment.production: [],
 }
 
-for branch in ["autoland", "beta", "esr115", "esr128", "release"]:
```

Comment:
We need to add `firefox-main`, can we use this script for that today before we remove it?

Reply:
@cgsheeh sure, just edit the script and include only `main` in the list of branches.

Reply:
You can also manually create it in the admin. This script was mostly to save time and avoid error because there were 5 repos.